lms/djangoapps/teams/static/teams/templates/team-member.underscore

```diff
@@ -1,7 +1,7 @@
 <li>
   <span class="team-member">
     <a class="member-profile" href="<%= memberProfileUrl %>">
```

Comment:
Yes.  In theory they should also be URL-escaped, and I'm not as clear on that, but HTML-escaping is right.